### ISSUE TITLE
Add a fifth column to the topics mega drop down[BGS: P2]

### ISSUE THREAD

**Issue by mrm-git** [MantaRayMedia/bgs#372]
# Done criteria
* [x] Fifth column needs widths adjusting in the styles

- - -
**parents:**

1. https://trello.com/c/bXfw3Bxu

┆Attachments: <a href="https:&#x2F;&#x2F;platform.harvestapp.com&#x2F;platform&#x2F;trello&#x2F;attachment&#x2F;5bf80cb7c844f61360b934b5&#x2F;LP3DM2o0">https:&#x2F;&#x2F;platform.harvestapp.com&#x2F;platform&#x2F;trello&#x2F;attachment&#x2F;5bf80cb7c844f61360b934b5&#x2F;LP3DM2o0</a>

**Comment by angievanhegan** [comment]
Angie to talk to Geraint about the design impact of adding a 5th column

**Comment by conferences-BGS** [comment]
@angievanhegan @russ-manta Can we see a mock up in staging of the 5th column? The request from Colin was 5 columns and no alternative appears to be offered above

**Comment by angievanhegan** [comment]
@cat-manta Do you think you can squeeze this in this week please? We're focussed on getting all these issues completed by 22 Nov. If you could create a mockup of the 5 columns, that would be really helpful. Thanks

**Comment by angievanhegan** [comment]
@conferences-BGS it might be useful to know that we specifically discussed with Steph when we built this, that having this mega drop down would be a commitment to keeping the number of categories down to four, although I appreciate that that's a challenge. 

However, here's a design showing what someone would see if they were using a laptop: https://projects.invisionapp.com/share/PENNUXHNXQY#/screens/332135499_Topics-Menu-V1-1_Laptop

And here's a design showing what someone would see if they had a widescreen: 
https://projects.invisionapp.com/share/PENNUXHNXQY#/screens/332135498_Topics-Menu-V1-1_Widescreen

The widescreen one obviously looks better, but we still think this is an enormous amount of information to present a user with and doesn't represent a fantastic user experience. We're not keen on either of these options. 

If there's absolutely no way to keep the categories to 4, another suggestion may be to lose the mega drop down and just have the standard drop down (e.g the standard navigation). Although, this would be a degradation of the current experience which allows people to scan the whole list of topics. 

In a nutshell, our advice is to do all you can to stick to 4 categories to make topics work as effectively as possible for your users. Let me know what you'd like to do.

**Comment by angievanhegan** [comment]
@conferences-BGS  It would be really great if you were able to let me have the content for the 5th category. Do you already have the name of the category and a list of the topics? Thanks

**Comment by angievanhegan** [comment]
@conferences-BGS we've discussed this internally and we think that option A is to try with 5 columns and edited titles (and perhaps some styling tweak). Plan B is to try stacking 2 columns. We'll let you know when you can put in the details for the 5th column.

**Comment by russ-manta** [comment]
@conferences-BGS you can now see this by hovering over topics in the menu on staging: http://bgs-staging.mrmdev2.co.uk/

**Comment by conferences-BGS** [comment]
@russ-manta the 5th Column needs to be moved so that it is placed immediately after Primary/Community (so middle) and should be titled ‘Cross-cutting Primary/Acute’

**Comment by russ-manta** [comment]
@conferences-BGS done on staging. FYI, this is all controlled via the main menu which you have complete control of positioning and naming for:
http://bgs-staging.mrmdev2.co.uk/admin/structure/menu/manage/main-menu

**Comment by conferences-BGS** [comment]
> @conferences-BGS done on staging. FYI, this is all controlled via the main menu which you have complete control of positioning and naming for:
> http://bgs-staging.mrmdev2.co.uk/admin/structure/menu/manage/main-menu
@russ-manta  can we add  a line divider if we keep 4 columns and move primary care below acute care?

**Comment by conferences-BGS** [comment]
Signed off - please move to live

**Comment by mrm-git** [comment]
➤ Angie Vanhegan commented:

@stevelacey @catjohnson2 Do you want to have a look at this and check you're happy with the presentation?

**Comment by stevelacey** [comment]
uh hi. I have no idea what this is or why you're @ ing me – but you can consider me happy with the implementation, whatever it is

**Comment by cat-manta** [comment]
Angie I think you @'d the wrong Steve. I don't love it on small screens but it's fine.
@angievanhegan @ican-icant

**Comment by mrm-git** [comment]
➤ Russ Urwin commented:

This is now live. Some topics need to be added to the new topics menu item .Cross cutting Primary/Acute.
